scripts/_common.sh

```diff
@@ -53,7 +53,7 @@ install_sources() {
         fi
         
         chown $synapse_user:root -R $final_path
-        sudo -u $synapse_user env PATH=$PATH pip3 install --upgrade 'cryptography>=3.4.7'
+        sudo -u $synapse_user env PATH=$PATH pip3 install --upgrade 'cryptography>=3.4.7' 'pyOpenSSL>=22.1.0'
```

Comment:
Should we append `attr attrs cryptography pyopenssl` and maybe remove `OpenSSL`

I had to apply following fix for upgrade and restore :
https://forum.yunohost.org/t/restoring-synapse-from-backup-python-6558-modulenotfounderror-no-module-named-attrs/22859/4

Reply:
It's different when you try to restore the backup or when you simply upgrade

Reply:
I'll add attr as it could be also in cause following the suggestions.

```suggestion
        sudo -u $synapse_user env PATH=$PATH pip3 install --upgrade 'cryptography>=3.4.7' 'pyOpenSSL>=22.1.0' 'attrs>=22.1.0'
```